docker/package/tezos_setup_wizard.py

```diff
@@ -165,15 +165,39 @@ def validate(self, input):
 suppress_warning_text = "TEZOS_CLIENT_UNSAFE_DISABLE_DISCLAIMER=YES"
 
 
+def exit_with_confirm(signum, frame):
+    signal.signal(signal.SIGINT, original_sigint)
+
+    try:
+        print()
+        if yes_or_no(
+            "You're about to quit the setup wizard. All the services and processes "
```

Comment:
There is a problem that during snapshot import or bootstrapping node process is actively spamming to the stdout and it's hard to notice that there was some prompt from the wizard, e.g. all you see when doing `Ctrl + C` when importing a snapshot is
```
Writing context: 1845K/4008K (46%) elements, 141MiB read^C
```